index.js

```diff
@@ -148,6 +226,120 @@ types[MAESTRO] = {
   }
 };
 
+types[ELO] = {
+  niceType: 'Elo',
+  type: ELO,
+  prefixPattern: new RegExp('^(' +
+    '[4-6]' +
+
+    '|' +
+
+    '4[035]|4[035]1' +
+    '|' +
+    '4011|40117|40117[89]' +
+    '|' +
+    '4312|43127|431274' +
+    '|' +
+    '438|4389|43893|438935' +
+    '|' +
+    '4514|45141|451416' +
+    '|' +
+    '457|457[36]|45739|45763|457393|45763[12]' +
+
+    '|' +
+
+    '50|50[69]' +
+    '|' +
+    '506[6-7]|50669|5067[0-7]|5067[0-6][0-9]|50677[0-8]' +
+    '|' +
+    '509[0-9]|509[0-9][0-9]|509[0-9][0-9][0-9]' +
+
+    '|' +
+
+    '6[235]|627|636|65[015]' +
+    '|' +
+    '6277|62778|627780' +
+    '|' +
+    '636[23]|63629|636297|63636|636368' +
+    '|' +
+    '650[0479]' +
+    '|' +
+    '6500[3-5]|65003[1-3]|65003[5-9]|65004[0-9]65005[01]' +
+    '|' +
+    '6504[0-3]|65040[5-9]|65041[0-9]' +
+    '|' +
+    '6505[4-9]|65054[1-9]|6505[5-8][0-9]|65059[0-8]' +
+    '|' +
+    '6507[0-2]|65070[0-9]|65071[0-8]|65072[0-7]' +
+    '|' +
+    '6509[0-7]|65090[1-9]|6509[1-6][0-9]|65097[0-8]' +
+    '|' +
+    '6516|6516[5-7]|65165[2-9]|6516[6-7][0-9]' +
+    '|' +
+    '6550|6550[0-5]|6550[01][0-9]|65502[1-9]|6550[3-4][0-9]|65505[0-8]' +
+  ')$'),
+  exactPattern: new RegExp('^(' +
+    // Elo only ranges
+    '4(31274|51416|57393)' +
+
+    '|' +
+
+    '50(' +
+      '4175' +
+      '|' +
+      '6699|67[0-6][0-9]|677[0-8]' + // 506699-506778
+      '|' +
+      '9[0-9][0-9][0-9]' + // 509000-509999
+    ')' +
+
+    '|' +
+
+    '627780' +
+
+    '|' +
+
+    '636(297|368)' +
+
+    '|' +
+
+    // Dual Branded with Visa
+    '4(0117[89]|38935|5763[12])' +
```

Comment:
For the bin started with 4, we found thats not a dual branded card. Those are old BINs that were utilized before ELO had established the partnership with Discover. So we just remap these bins to Elo only card.

Reply:
Okay, I'll update the comment.